build.sbt

```diff
@@ -33,7 +33,7 @@ val netty4StaticSslVersion: String =
 val nettyVersionInfo = settingKey[String]("A setting reference for printing the netty version info")
 
 // zkVersion should be kept in sync with the 'util-zk' dependency version
-val zkVersion = "3.5.0-alpha"
+val zkVersion = "3.5.6"
```

Comment:
Could you also bump the util-zk version in the dependency? https://github.com/twitter/util
thanks!

Reply:
Hey, don't worry about this, we update it along with this PR. We plan to merge this next week asap, thank you!

Reply:
Thanks!